manifests/setup/node_manager.pp

```diff
@@ -124,7 +124,9 @@
         'database_host' => $puppetdb_database_host,
       },
       'puppet_enterprise::profile::master'   => {
-        'puppetdb_host' => ['${trusted[\'certname\']}', $internal_compiler_b_pool_address].filter |$_| { $_ }, # lint:ignore:single_quote_string_with_variables
+        # lint:ignore:single_quote_string_with_variables
```

Comment:
@reidmv  Should we remove these commented lines? (Ln. 127 and Ln. 129)

Reply:
These are needed to make `pdk validate` pass cleanly. They are [puppet-lint control comments](http://puppet-lint.com/controlcomments/). It's a little ugly, but I think they are worthwhile to let the linter know that what's done on the next line is intentional.

Reply:
aah so nothing to do there, thanks for the context 👍

Reply:
Oh, and I replaced the end-of-line comment with the block-form because the end-of-line comment made the content run afoul of another lint issue, the 140 character line limit 🤦